packages/x-charts-pro/src/internals/plugins/useChartProZoom/useChartProZoom.selectors.ts

```diff
@@ -49,3 +51,49 @@ export const selectorChartCanZoomIn = createSelector(
     });
   },
 );
+
+const selectorChartBrushState: ChartRootSelector<UseChartProZoomSignature, 'zoomBrushState'> = (
+  state,
+) => state.zoomBrushState;
+
+export const selectorChartBrushStartX = createSelector(
+  [selectorChartBrushState],
+  (brushState) => brushState.start?.x ?? null,
+);
+
+export const selectorChartBrushStartY = createSelector(
+  [selectorChartBrushState],
+  (brushState) => brushState.start?.y ?? null,
+);
+
+export const selectorChartBrushCurrentX = createSelector(
+  [selectorChartBrushState],
+  (brushState) => brushState.current?.x ?? null,
+);
+
+export const selectorChartBrushCurrentY = createSelector(
+  [selectorChartBrushState],
+  (brushState) => brushState.current?.y ?? null,
+);
+
+export const selectorChartBrushConfig = createSelector([selectorChartSeriesProcessed], (series) => {
+  let hasHorizontal = false;
+  let hasScatter = false;
+  if (series) {
+    Object.entries(series).forEach(([seriesType, seriesData]) => {
+      if (Object.values(seriesData.series).some((s) => s.layout === 'horizontal')) {
+        hasHorizontal = true;
+      }
+      if (seriesType === 'scatter' && seriesData.seriesOrder.length > 0) {
+        hasScatter = true;
+      }
+    });
+  }
+  if (hasScatter) {
+    return 'xy';
```

Comment:
Shouldn't it apply to xy only when there is at least one axis in each direction that has `zoom: true`? 

If I have a scatter chart where I can only zoom in the x-axis, shouldn't we render a full-height rectangle?